src/hearings/containers/request-hearing/date-priority-hearing/date-priority-hearing.component.ts

```diff
@@ -68,24 +99,42 @@ export class DatePriorityHearingComponent implements OnInit {
   public initForm(): void {
     this.priorityForm = this.formBuilder.group({
       durationLength: this.formBuilder.group({
-        hours: [],
-        minutes: []
+        hours: ['', [this.validatorsService.numberMinMaxValidator(HearingDatePriorityConstEnum.MinHours, HearingDatePriorityConstEnum.MaxHours)]],
+        minutes: ['', [this.validatorsService.numberMultipleValidator(HearingDatePriorityConstEnum.MinutesMuliplier)]]
+      }, { validator: this.validatorsService.minutesValidator(HearingDatePriorityConstEnum.TotalMinMinutes, HearingDatePriorityConstEnum.TotalMaxMinutes) }),
+      specificDate: ['', Validators.required],
+      firstHearing: this.formBuilder.group({
+        firstHearingDate_day: [],
+        firstHearingDate_month: [],
+        firstHearingDate_year: [],
       }),
-      firstHearingDate_day: [],
-      firstHearingDate_month: [],
-      firstHearingDate_year: [],
-      earliestHearingDate_day: [],
-      earliestHearingDate_month: [],
-      earliestHearingDate_year: [],
-      latestHearingDate_day: [],
-      latestHearingDate_month: [],
-      latestHearingDate_year: [],
-      hearingAvailability: [],
+      dateRangeHearing: this.formBuilder.group({
+        earliestHearing: this.formBuilder.group({
+          earliestHearingDate_day: [],
+          earliestHearingDate_month: [],
+          earliestHearingDate_year: [],
+        }),
+        latestHearing: this.formBuilder.group({
+          latestHearingDate_day: [],
+          latestHearingDate_month: [],
+          latestHearingDate_year: [],
+        }),
+      }),
+      priority: ['', Validators.required]
     });
   }
 
   public showDateAvailability(): void {
-    this.checkedHearingAvailability = this.priorityForm.get('hearingAvailability').value;
+    this.checkedHearingAvailability = this.priorityForm.controls.specificDate.value;
+    this.firstHearingFormGroup.clearValidators();
+    this.dateRangeHearingFormGroup.clearValidators();
+    if (this.checkedHearingAvailability === 'Yes') {
```

Comment:
can this be enum type put in src/hearings/hearings.enum
```suggestion
export enum RadioOptions {
 YES = 'Yes',
 NO = 'No',
 CHOOSE_DATE_RANGE = 'Choose a date range'
}
```